### PR TITLE
Filter according to name and id

### PR DESCRIPTION
**Pull request by laurianed**
name: contains, starts with
id: lower than, greater than

### CODE REVIEW

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/GraphQLQueries.java [hunk 1]
```diff
@@ -96,6 +83,7 @@ public Query getRevisionAndjobsInfoQuery(final String user, final boolean pendin
             jobsBuilder.input(input);
 
             Query.Builder queryBuilder = new Query.Builder().query(jobsBuilder.build().getQueryString());
+            LOGGER.log(Level.SEVERE, queryBuilder.build().getQuery());
```
Comment (tobwiens) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/GraphQLQueries.java]: Did you forget this log output?

Comment (laurianed) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/GraphQLQueries.java]: Yes

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/GraphQLQueries.java [hunk 2]
```diff
@@ -135,29 +122,81 @@ public Query getRevisionAndjobsInfoQuery(final String user, final boolean pendin
...
+                            id = getValue(id, value);
+                            break;
+                        case GREATER_THAN_OR_EQUAL_TO:
+                            input.greaterThanId(value);
+                            break;
+                        case LESS_THAN_OR_EQUAL_TO:
+                            input.lowerThanId(value);
+                            break;
+                        default:
+                            ;
```
Comment (MaelAudren) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/GraphQLQueries.java]: Can you add a break for the default case in order to keep consistency in the code

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/GraphQLQueries.java [hunk 3]
```diff
@@ -135,29 +122,81 @@ public Query getRevisionAndjobsInfoQuery(final String user, final boolean pendin
...
+                            name = getValue(name, value);
+                            break;
+                        case CONTAINS:
+                            name = getValue(name, "*" + value + "*");
+                            break;
+                        case STARTS_WITH:
+                            name = getValue(name, value + "*");
+                            break;
+                        default:
+                            ;
```
Comment (MaelAudren) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/GraphQLQueries.java]: Can you add a break for the default case in order to keep consistency in the code

**Pull request merged**